packages/core/src/base/staticClient.ts

```diff
@@ -13,7 +13,10 @@ export { DefaultCache, Response };
 import { Resource } from "./resource";
 import { BaseClient } from "./client";
 import { sdkLogger } from "./sdkLogger";
+import pkg from "../../package.json";
 
+// Version is from @commerce-apps/core, but it will always match commerce-sdk
```

Comment:
+1 for calling this out..

Reply:
pls add or update a test in integration to verify this new header matching the commerce-sdk version

Reply:
I have added a test.